secrets/secrets.go

```diff
@@ -20,16 +18,13 @@ type Secrets struct {
 
 // New parses secrets file.
 func readFile(secretsFile string) ([]byte, error) {
-	logger.Plain.Info(
-		"Get secrets",
-		zap.String("secretsFile", secretsFile),
-	)
+	logger.Sugar.Infof("Get secrets %s", secretsFile)
```

Comment:
why Sugar here, when in all the others it is being replaced?

Reply:
an extra argument would be added to secrets.New() and that change is out of scope for this PR.

but yes you are right - we eventually make such a change - not in this PR though